CommonWallet/Classes/Common/Fee/FeeCalculationFactory.swift

```diff
@@ -0,0 +1,75 @@
+/**
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
+* SPDX-License-Identifier: GPL-3.0
+*/
+
+import Foundation
+import IrohaCommunication
+
+enum FeeType: String, Codable, Equatable {
+    case factor = "FACTOR"
+    case fixed = "FIXED"
```

Comment:
Have it been documented?

Reply:
Yes, I have added this to Capital API documentation.